pyproject.toml

```diff
@@ -186,3 +150,68 @@ replace = "__version__ = '{new_version}'"
 [build-system]
 requires = ['setuptools', 'wheel']
 build-backend = 'setuptools.build_meta'
+
+[tool.ruff]
+preview = true
+line-length = 100
+indent-width = 4
+src = ["ctgan"]
+exclude = [
+    "docs",
+    ".tox",
+    ".git",
+    "__pycache__",
+    ".ipynb_checkpoints"
+]
+
+[tool.ruff.lint]
+select = [
+    # Pyflakes
+    "F",
+    # Pycodestyle
+    "E",
+    "W",
+    # pydocstyle
+    "D",
+    # isort
+    "I001",
+    # print statements
+    "T201",
+    # pandas-vet
+    "PD"
+]
+ignore = [
+    "E501",
+    # pydocstyle
+    "D100",
+    "D101",
+    "D102",
+    "D103",
+    "D104",
+    "D105", # Missing docstring in magic method
+    "D107",  # Missing docstring in __init__
```

Comment:
will all other docstrings be enforced?

Reply:
I made it so pydocstyle is ignored in the `tests` directory, but enforced elsewhere:
https://github.com/sdv-dev/CTGAN/pull/389/commits/1b6a9fc2bd6a247764c67b4844b817116a2d85c3